feat(appellate): Adds logic to upload the Docket report from ACMS (#372)
freelawproject/recap-chrome: CHANGES.md

Features:
 - Introduces a new upload type for Dockets from ACMS and enhances code maintainability by refactoring common upload logic into reusable functions.([#368](https://github.com/freelawproject/recap-chrome/pull/371))

>  - Updates the AppellateDelegate class to upload Docket reports from ACMS

Reviewer: This should be written for humans, not coders. Perhaps: 

```suggestion
 - Docket reports from ACMS are now uploaded to CourtListener.
```

Author: got it! I'll update the chagelog.
